Snake/BusinessLayer/GameSnake.cs

```diff
@@ -38,12 +41,12 @@ public List<Point> createFirstSnake(int width, int height, int newLength)
 
 
         public List<Point> snakeMove(int direction, Boolean hasEaten)
```

Comment:
there are standard conventions in .Net for naming so i suggest you identify these and all follow them. i sent a link to style guide the other day that we use. Pick your own if you like but do follow established conventions of the language.
For example methods and properties use pascal case.